rialto_airflow/harvest/distill.py

```diff
@@ -183,87 +184,3 @@ def _apc_oa_dataset(dim_json, context):
             return cost
 
```

Comment:
This is the code that was extracted to `rialto_airflow.distiller` and refactored along the way.